packaging/cpackconfig_common.cmake

```diff
@@ -1,13 +1,19 @@
 cpack_add_component(_cura
-                    DISPLAY_NAME "Ultimaker Cura"
-                    Description "Ultimaker Cura Executable and Data Files"
-                    REQUIRED
+		DISPLAY_NAME "Ultimaker Cura ${CURA_BUILDTYPE}"
```

Comment:
Will a regular build be called "Ultimaker Cura " (note the extra space)?

Reply:
I was a bit hesitant about that one, too. The only place the DISPLAY_NAME is used is during installation, where the user reads the license, etc., in cases like: "Installing Ultimaker Cura " or "Ultimaker Cura  Setup" (2 spaces in the last example). After checking it though, it is not really obvious and easily visible. There is a workaround of course, if you don't like it (and I agree it's not the "best" way to do it).

Reply:
If it's only visible there i'll accept it